src/oci/image/manifest.rs

```diff
@@ -28,12 +28,16 @@
 //  }
 //}
 
-use std::collections::HashMap;
+use std::collections::HashMap as Map;
 
 use serde::{Deserialize, Serialize};
 
 use crate::oci::Hash;
 
+pub const MANIFEST_MIME: &str = "application/vnd.docker.distribution.manifest.v2+json";
```

Comment:
I believe you can remove the type. It should be inferred.

Reply:
Nop, `rustc` is not happy without the type :
> [rustc] [E] missing type for `const` item

Reply:
Oh, right. 

https://doc.rust-lang.org/std/keyword.const.html